golem/payload/vm.py

```diff
@@ -52,6 +52,10 @@ class BaseVmPayload(Payload, ABC):
         "golem.srv.comp.vm.package_format", default=VmPackageFormat.GVMKIT_SQUASH
     )
 
+    debit_notes_accept_timeout: int = prop(
```

Comment:
okay, I'm unconvinced that this should be part of the payload ... this is defnitely part of the demand but similarly to the payment-related properties, it's something common to the whole configuration of the requestor agent and thus, is independent from what we'd want from a particular provider node ...